src/drive/web/modules/viewer/Footer/BottomSheet.jsx

```diff
@@ -96,6 +95,15 @@ const BottomSheetWrapper = ({
       fullHeight={false}
       onIndexChange={snapIndex => handleOnIndexChange(snapIndex)}
       styles={{ root: styles.root }}
+      threshold={0}
+      // springConfig doc : https://www.react-spring.io/docs/hooks/api
+      springConfig={{
+        tension: 165,
+        friction: 17,
+        clamp: true
+      }}
```

Comment:
Can't you use flag for those values? And make it writable in the Settings Views?

Reply:
(If you need to test different values for those settings, maybe https://cocopon.github.io/tweakpane/ can be useful)

Reply:
Merci pour le lien je garde sous le coude. Je suis parti sur des input classiques dans notre cas : j'ai vu ton message un peu tardivement en cours de dev + pas de nécessité ici d'avoir un lib supplémentaire je pense.